src/System.Runtime.WindowsRuntime.UI.Xaml/src/System/Windows/Duration.cs

```diff
@@ -255,18 +255,7 @@ public Duration Add(Duration duration)
 
         public override bool Equals(Object value)
         {
-            if (value == null)
-            {
-                return false;
-            }
-            else if (value is Duration)
-            {
-                return Equals((Duration)value);
-            }
-            else
-            {
-                return false;
-            }
+            return value is Duration && Equals((Duration)value);
```

Comment:
Take solace in the fact that some of the other changes in this PR are indeed turning `is` + cast into `as` + null-check.